portfolio/src/main/webapp/index.html

```diff
@@ -7,13 +7,31 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>Katya's Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    <p>Click <a href="Kite_gallery.html">here</a> to see my kite gallery!</p>
+        <div id="content">
+        <h1>Katya's Portfolio</h1>
+        <text id = "intro"> Hi! My name is Katya...</text> 
```

Comment:
please remove trailing whitespaces on text and on attributes as well
https://google.github.io/styleguide/htmlcssguide.html#Trailing_Whitespace